test_utils/fake_enrollment_api.py

```diff
@@ -151,6 +164,41 @@ def get_course_details(course_id):
         return None
 
 
+def lms_enroll_user_in_course(
```

Comment:
this may be a very stupid question, but where/how are we using this fake enroll method?
I don't see anything importing from `fake_enrollment_api` nor even `test_utils`

Reply:
haha good q let me check if I do actually use it still. I was using it once but maybe left this behind after a nother refactor.. brb

Reply:
yeah thanks for the catch I removed after verifying it's no longer used